src/ml/neural_net/mps_layer_helper.mm

```diff
@@ -0,0 +1,90 @@
+#include <ml/neural_net/mps_layer_helper.h>
+#include <ml/neural_net/mps_weight.h>
+#include <ml/neural_net/mps_layer_conv_padding.h>
+
+@implementation TCMPSLayerHelper
+
++ (MPSCNNFullyConnectedNode *) createFullyConnected:(MPSNNImageNode *)inputNode
+                               inputFeatureChannels:(NSUInteger)inputFeatureChannels
+                              outputFeatureChannels:(NSUInteger)outputFeatureChannels
+                                        inputHeight:(NSUInteger)inputHeight
+                                         inputWidth:(NSUInteger)inputWidth
+                                            weights:(float *)weights
+                                             biases:(float *)biases
+                                              label:(NSString *)label
+                                      updateWeights:(bool)updateWeights
+                                             device:(id<MTLDevice>)dev
+                                          cmd_queue:(id<MTLCommandQueue>) cmd_q {
+  
+  turi::neural_net::OptimizerOptions optimizerOptions;
+
+  TCMPSConvolutionWeights *FullyConnectedDataLoad = [[TCMPSConvolutionWeights alloc] initWithKernelWidth:inputWidth
+                                                                                  kernelHeight:inputHeight
+                                                                          inputFeatureChannels:inputFeatureChannels
+                                                                         outputFeatureChannels:outputFeatureChannels
+                                                                                    neuronType:MPSCNNNeuronTypeNone
+                                                                                       strideX:1
+                                                                                       strideY:1
+                                                                                       neuronA:0.0f
+                                                                                       neuronB:0.0f
+                                                                        kernelParamsBinaryName:[label UTF8String]
+                                                                                        device:dev
+                                                                                     cmd_queue:cmd_q
+                                                                               init_weight_ptr:weights
+                                                                                 init_bias_ptr:biases
+                                                                              optimizerOptions:optimizerOptions];
+        
+  MPSCNNFullyConnectedNode* FullyConnectedNode = [MPSCNNFullyConnectedNode nodeWithSource:inputNode
```

Comment:
Nit: variable names should not start with a capital letter

Reply:
changed to lowercase.